public_app/pages/AllMovies.js

```diff
@@ -0,0 +1,41 @@
+import React        from 'react';
+import Nav          from '../components/Nav';
+import LogOut       from '../components/LogOut';
+import MovieItem    from '../components/MovieItem';
+import axios        from 'axios';
+
+var Link = require ('react-router-dom').Link;
+
+class AllMovies extends React.Component {
+    constructor (props) {
+        super(props);
+        this.state = {
+            items: []
+        };
+        axios.get("api/movies").then((response) => {
+            this.setState({
+                items: response.data
+            });
+            console.log(response);
+        });
+
+    }
+    render (){
+        return (
+            <div className="all-movies-container">
+                <Link to={'/login'}>
+                    <LogOut />
+                </Link>
+                <Nav />
+                <div className="">
+                    <div className='all-img-view'>
+                        {this.state.items.map((item) => {
+                            return <MovieItem key={item.Title} id={item._id} src={item.Poster}/>;
```

Comment:
Can again be a function that returns the JSX without `return` keyword

`this.state.items.map(item => <MovieItem... />`